hydrosdk/model.py

```diff
@@ -151,7 +154,9 @@ def __upload(self, cluster):
             "runtime": {"name": self.runtime.name,
                         "tag": self.runtime.tag,
                         "sha256": self.runtime.sha256},
-            "contract": contract_to_dict(self.contract)
+            "contract": contract_to_dict(self.contract),
+            "installCommand": self.install_command,
+            "metadata": self.metadata
```

Comment:
server expects `metadata` keys and values to be of type `str`
Need to add check/conversion before uploading it

Reply:
Fixed